### PR TITLE
Nav and grid

### PR DESCRIPTION
**Pull request by irajbhinde**
Hey! Please review my code & let me know if any changes could be made.

Added Components -

1. Simple Nav Bar
2. Grid - Grid w Two Items & Grid w Three Items

GIFY - 
nav_grid

### CODE REVIEW

File: Components/Grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.grid {
+  display: grid;
+  gap: 1rem;
+  width: 50rem;
+  height: 5rem;
+}
+
+.grid_w_two-items {
+  grid-template-areas: "ItemOne ItemTwo";
```
Comment (shlok30) [inline Components/Grid/grid.css]: Hey,
Your implementation and logic looks on point, but the idea of Grid system is to provide a Row and Column sort of matrix.
Your examples here are only showing a single row with multiple columns which can already be easily achieved by using Flex alone.
I'd suggest you to modify grid-template-areas to accomodate multiple rows as well

File: Components/Grid/grid.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+.grid_w_three-items{
+    grid-template-areas: "ItemOne ItemTwo ItemThree";
```
Comment (shlok30) [inline Components/Grid/grid.css]: Same suggesting here as well

File: Components/Nav/nav.html [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+            </code>
+        </pre>
+        <div class="content-example">
+            <nav>
+                <div class="nav-bar">
+                    <div class="left-navbar">
+                      <h2><a href="/"> Title </a></h2>
+                    </div>
+                    <div class="search-bar flex_r">
+                      <input
```
Comment (shlok30) [inline Components/Nav/nav.html]: Add some styling to input element as currently its completely blending with the nav background

File: Components/Nav/nav.css [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  align-items: center;
+}
+
+.search-bar input {
+  display: flex;
+  align-items: center;
+  border: 1px solid #f5f5f6;
+  width: 30rem;
+  height: 2.5rem;
+  background-color: #f5f5f6;
```
Comment (chanchal-panpaliya) [inline Components/Nav/nav.css:25]: use var() for color code .

File: Components/Nav/nav.html [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+                  <li><a href="/Components/Ratings/ratings.html">Ratings</a></li>
+                  <li><a href="/Components/Snackbar/snackbar.html">Snackbar</a></li>
+                  <li><a href="/Components/Chips/chips.html">Chips</a></li>
+                  <li><a href="/Components/Nav/nav.html">Navigation Bar</a></li>
+                </ul>
+            </div>
+        </div>
+        <div class="content-bar">
+            <div class="content-title">Navigation Bar</div>
+        <div class="content-subtitle border_to_content">
```
Comment (chanchal-panpaliya) [inline Components/Nav/nav.html:58]: format your code.

**Review by chanchal-panpaliya — COMMENTED** [review]
+1 shlock - You can work on the suggestions above . Also , You can format your code for sanity , use var(--colorcode) for color code.

**Pull request merged**